ui/lib/sync/addon/components/secrets/page/destinations/destination/secrets.hbs

```diff
@@ -3,4 +3,85 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<Secrets::DestinationHeaderTabsToolbar @destination={{@destination}} />
\ No newline at end of file
+<Secrets::DestinationHeader @destination={{@destination}} />
+
+{{#if @associations.meta.filteredTotal}}
+  <div class="has-bottom-margin-s">
+    {{#each @associations as |association index|}}
+      <ListItem as |Item|>
+        <Item.content>
+          <div>
+            <span data-test-association-name={{index}}>
+              {{association.id}}
+            </span>
+            <Hds::Badge @text={{association.syncStatus}} @color="neutral" data-test-association-status={{index}} />
+            <div>
+              <code class="has-text-grey is-size-8" data-test-association-updated={{index}}>
+                last synced on
+                {{date-format association.updatedAt "MMMM do yyyy, h:mm:ss a"}}
+              </code>
+            </div>
+          </div>
+        </Item.content>
+
+        <Item.menu as |Menu|>
+          {{#if (or association.setAssociationPath.isLoading association.removeAssociationPath.isLoading)}}
+            <li class="action">
+              <LoadingDropdownOption />
+            </li>
+          {{else}}
+            <li>
+              <Hds::Button
+                @text="Sync now"
+                class="link"
+                disabled={{not association.canSync}}
```

Comment:
I think we're moving away from disabling dropdown options and just hiding them if the user doesn't have permission

Reply:
I think I brought this up a while ago and was told to disable in menus, otherwise it allows the scenario for a user to open a menu with nothing in it. We could then hide the menu trigger as well but it's not something that we've done as far as I can tell.